gtests/net/mptcp/add_addr/add_addr4_port_client.pkt

```diff
@@ -18,5 +18,5 @@
 +0.205  fcntl(3, F_SETFL, O_RDWR) = 0   // set back to blocking
 
 // ADD_ADDRESS with port
-+0.0      <   .  1:1(0)  ack 1  win 256                                                         <add_address address_id=12 addr[ep=inet_addr("198.51.100.2")] port=4321 hmac=auto, dss dack4=1 dll=0 nocs, nop, nop>
++0.0      <   .  1:1(0)  ack 1  win 256                                                         <add_address address_id=12 addr[ep=inet_addr("198.51.100.2")] port=4321 hmac=auto, nop, nop>
```

Comment:
@pabeni this is an inbound packet, so I expect that the test will pass anyway even if we don't remove the the DSS sub-option here.  Correct? IOW, does the kernel still process an inbound `ADD_ADDR` + DSS  ?